shell/platform/fuchsia/flutter/session_connection.cc

```diff
@@ -173,7 +186,7 @@ fml::TimePoint DefaultSessionConnection::SnapToNextPhase(
   }
 }
 
-DefaultSessionConnection::DefaultSessionConnection(
+SessionConnection::SessionConnection(
```

Comment:
Since we're already renaming this, let's change it to GfxSessionConnection so the Flatland* one makes sense.